boards/arm/ngultra_bring_up/ngultra_bring_up_defconfig

```diff
@@ -0,0 +1,8 @@
+CONFIG_SOC_NGULTRA=y
+CONFIG_SOC_SERIES_NANOXPLORE_NGULTRA=y
+CONFIG_BOARD_NGULTRA_BRING_UP=y
+CONFIG_CLOCK_CONTROL=n
```

Comment:
Any reason you need to explicitly set `CONFIG_CLOCK_CONTROL=n`?

Reply:
No, I removed the config